plugin.py

```diff
@@ -30,19 +36,19 @@ def start_julia_repl(window: sublime.Window, focus: bool) -> None:
     window.run_command("terminus_open", {
         "cmd": cmd,
         "cwd": "${file_path:${folder}}",
-        "panel_name": JULIA_REPL_NAME,
+        "title": panel and "" or JULIA_REPL_NAME,
```

Comment:
the `"title"` argument is then what is returned upon calling `view.name()`

Reply:
This expression will always evaluate to just `JULIA_REPL_NAME`.

Reply:
whops, `not panel and JULIA_REPL_NAME or None` should work

Reply:
I guess the value for "title" doesn't matter at all, if it is opened in a panel. So you could just leave it as `"title": JULIA_REPL_NAME`. The "terminus_open" command seems to decide whether a panel or tab should be used, based only on the "panel_name" value.